Sorry: IndentationError: unexpected indent (serial_temp_reader.py, line 52)
